content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -78,5 +78,5 @@ very small. Please be aware of this risk when setting a non-zero TTL.
 
 * [Clean up Jobs automatically](/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically)
 
-* [Design doc](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/0026-ttl-after-finish.md)
+* [Design doc](https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/592-ttl-after-finish/README.md)
```

Comment:
verified that new link fixes the previous broken link and points to the correct KEP! Great job!